pykickstart/commands/repo.py

```diff
@@ -170,44 +170,54 @@ def __str__(self):
 
     def _getParser(self):
         op = KSOptionParser(prog="repo", description="""
-.. versionadded:: Fedora6
-
-Configures additional yum repositories that may be used as sources for
-package installation. Multiple repo lines may be specified. By default,
-anaconda has a configured set of repos taken from /etc/anaconda.repos.d
-plus a special Installation Repo in the case of a media install. The
-exact set of repos in this directory changes from release to release and
-cannot be listed here. There will likely always be a repo named
-"updates".
-
-Note: If you want to enable one of the repos in /etc/anaconda.repos.d
-that is disabled by default (like "updates"), you should use --name= but
-none of the other options. anaconda will look for a repo by this name
-automatically. Providing a baseurl or mirrorlist URL will result in
-anaconda attempting to add another repo by the same name, which will
-cause a conflicting repo error.""")
+                            .. versionadded:: Fedora6
+
+                            Configures additional yum repositories that may be
+                            used as sources for package installation. Multiple
+                            repo lines may be specified. By default, anaconda
+                            has a configured set of repos taken from
+                            /etc/anaconda.repos.d plus a special Installation
+                            Repo in the case of a media install. The exact set
+                            of repos in this directory changes from release to
+                            release and cannot be listed here. There will
+                            likely always be a repo named "updates".
+
+                            Note: If you want to enable one of the repos in
+                            /etc/anaconda.repos.d that is disabled by default
+                            (like "updates"), you should use --name= but none
+                            of the other options. anaconda will look for a repo
+                            by this name automatically. Providing a baseurl or
+                            mirrorlist URL will result in anaconda attempting
+                            to add another repo by the same name, which will
+                            cause a conflicting repo error.""")
         op.add_argument("--name", required=True, help="""
```

Comment:
@bcl - how does the new formatting look like ? I will format all new descriptions in this manner and then go back and update the others as well